mobile_verifier/src/radio_threshold.rs

```diff
@@ -376,13 +375,13 @@ pub async fn verified_radio_thresholds(
 ) -> Result<VerifiedRadioThresholds, sqlx::Error> {
     let mut rows = sqlx::query_as::<_, RadioThreshold>(
         "SELECT hotspot_pubkey, cbsd_id
```

Comment:
do we need to select the cbsd_id field any longer here?

Reply:
Done